lib/ui/views/onboarding_view.dart

```diff
@@ -14,68 +16,82 @@ class _OnBoardingViewState extends State<OnBoardingView> {
   void initState() {
     super.initState();
     currentIndexPage = 0;
-    pageLength = 3;
+    pageLength = 6;
```

Comment:
don't hardcode, declare array of pages elsewhere. use `len()` or `.length` or whatever. you should be able to comment an entry from the array and not break the app.